DAQ/src/CrvDigisFromFragments_module.cc

```diff
@@ -103,41 +106,66 @@ void CrvDigisFromFragments::produce(Event& event)
 
     for(size_t iDataBlock = 0; iDataBlock < CRVDataDecoder.block_count(); ++iDataBlock)
     {
-      if(_diagLevel>0) std::cout << "iSubEvent/iDataBlock: " << iSubEvent << "/" << iDataBlock << std::endl;
-
       auto block = CRVDataDecoder.dataAtBlockIndex(iDataBlock);
       if(block == nullptr)
       {
-        std::cerr << "Unable to retrieve block in " << std::endl;
+        std::cerr << "iSubEvent/iDataBlock: " << iSubEvent << "/" << iDataBlock << std::endl;
```

Comment:
If this module is to be used in the TDAQ, `TLOG(TLVL_ERROR)` is preferred for compatibility with messaging systems. For purely Offline use, I'd still recommend using at least mf::LogError (though TLOG should be fully compatible)

Reply:
Using TLOG also obviates the need for _diagLevel-style constructs, as TRACE messages can be enabled/disabled dynamically at runtime without altering module configuration.

Reply:
This module will only be used in Offline. Most of the code in Offline uses cout/cerr/diagLevel while TLOG and mf::LogError are used only at a few places in Offline.